### PR TITLE
The Ark of the Clockwork Justicar can now be constructed even if Ratvar's rise is not an objective

### PR DESCRIPTION
**Pull request by X-TheDark**
Finishes porting https://github.com/tgstation/tgstation/pull/19864 by Joan/ChangelingRain (we had a lot of this already, but due to merge conflicts, it was never ported over completely)

:cl: Joan
rscadd: The Ark of the Clockwork Justicar can now be constructed even if Ratvar's rise is not the objective.
experiment: Accordingly, its behavior is different; instead of summoning Ratvar, it will convert a massive amount of the station, and should effectively instantly complete the clockwork cult's objectives.
/:cl:

Special request by @AdamElTablawy

### CODE REVIEW

**Comment by X-TheDark** [comment]
Also, I'm fully aware that some of these lines are in plain English, while most everything else is still in Caesar cipher. I'm not fixing either one of these group of lines to be consistent with the other, sorry.

**Comment by ShadowDeath6** [comment]
Meh. It's going to turn clock cult into the same boring turtle theme that we get every round that they have summon on. Not a huge fan.
